src/AerospikeCache.php

```diff
@@ -0,0 +1,130 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+
+class AerospikeCache extends AbstractAdapter
+{
+    private const WRAPPER_NAME = 'data';
+
+    /** @var \Aerospike */
+    private $aerospike;
+
+    /** @var string */
+    private $namespace;
+
+    /** @var string */
+    private $set;
+
+    public function __construct(
+        \Aerospike $aerospike,
+        string $namespace,
+        string $set,
+        string $cacheNamespace = '',
```

Comment:
I'm confused about `$namespace` and `$cacheNamespace`. What is the difference? What does it mean? Could you please add PHP doc to make it clear (or rename the variables, so it is obvious even without the PHP doc)?

Reply:
Well the name was originaly $cacheNamespace and $aerospikeNamespace, but i changed it on suggestion that $namespace in class AerospikeCache is obviously $aerispikeNamespace and the aerospike part is not neccesary. So i am not sure what to do now. The need for a comment shows a need for better variable name, but on the other hand it is not desirable as it is redundant information by others.

Reply:
Because this uses interface of AbstractAdapter (where `$namespace` is also used), I'd not agree with the statement "$namespace in class AerospikeCache is obviously $aerospikeNamespace".

Also the phpdoc might help with clarity - we can't include all of the information into variable name.
```
/**
 * @param string $aerospikeNamespace Namespace is a top level container for data in Aerospike - something similar to database name
 * @param string $cacheNamespace The string prefixed to the keys of the items stored in this cache
 */
```

Reply:
added comments and changed name to aerospikeNamespace